src/modules/api_score.js

```diff
@@ -0,0 +1,28 @@
+import Body from './body_class.js';
+
+async function sendScore(user, score, url) {
+  const requestURL = url;
+  const request = new Request(requestURL);
+  const data = new Body(user, score);
+
```

Comment:
- [ ] Kindly use the arrow function instead of regular functions. [why you should arrow functions.](https://betterprogramming.pub/why-javascript-arrow-functions-are-useful-aa023bad403a)